share/validate-product-config/product-config-schema.rnc

```diff
@@ -411,47 +506,65 @@ ds.deliverable_subdeliverable =
   }
 
 ds.dc =
+  ## Reference to DC file
   element dc {
     ds.type.dcfile
   }
 
 # Unfortunately, this way, I can't force people to choose at least one format
 # they actually want to build.
 ds.format =
+  ## Allows enabling/disabling document formats, at least one format must be enabled
   element format {
+    ## Whether to enable multi-page HTML builds
     attribute html { xsd:boolean }?,
+    ## Whether to enable single-page HTML builds
     attribute single-html { xsd:boolean }?,
+    ## Whether to enable PDF builds
     attribute pdf { xsd:boolean }?,
+    ## Whether to enable EPUB builds
     attribute epub { xsd:boolean }?
   }
 
 ds.param =
+  ## Custom XSLT parameter
   element param {
+    ## Name of the XSLT parameter
     attribute name { ds.type.xsltparam },
     text
   }
 
 
 ds.subdeliverable =
+  ## Individual document within a DocBook set as identified by their XML ID (default language)
   element subdeliverable {
     attribute category { ds.type.idmulti }?,
+    ## How to display the document title on the product version navigational page, a combination of the following values:
```

Comment:
If I'm not wrong, technically, you can only select *one* value, not a combination. They are mutually exclusive. This phrase sound a bit strange to me. Maybe just remove this to avoid confusion:

```suggestion
    ## How to display the document title on the product version navigational page:
```

Reply:
No, you can define a combination in which:

* `title` must appear
* `product` and `docset` are mutually exclusive.

That is, the following values are allowed here: `title`, `title subtitle`, `title subtitle product`, `title product`, `title subtitle docset`, or `title docset`. The default, as mentioned in the docs already, is `title subtitle`.

Reply:
Ahh I see. In that sense, combination is the correct term. I retract my comment. :wink: